packages/material/scss/treeview/_theme.scss

```diff
@@ -0,0 +1,31 @@
+
+@import "~@progress/kendo-theme-default/scss/treeview/theme";
+
+@include exports('treeview/theme/material') {
+    $treeview-icon-size: 24px !default;
+
+    .k-treeview {
+        .k-mid {
+            &:focus,
+            &:hover {
+                cursor: pointer;
+                background: $hovered-bg;
+            }
+            &:focus,
+            &:active {
+                outline: 0;
+                color: $accent;
+            }
+        }
+        .k-in {
+            &.k-state-selected {
+                color: $accent;
+                background-color: inherit;
+            }
+        }
+
+        .k-icon {
+            font-size: $treeview-icon-size;
```

Comment:
This rule should not be necessary at this time, given the [comment on icons here](https://github.com/telerik/kendo/issues/8250#issuecomment-377214034). If this is intentionally added only for the expand/collapse icons, the selector should be

```scss
.k-top,
.k-mid,
.k-bot {
   > .k-icon { ... }
}
```

(otherwise all icons in the TreeView will become large)